beta/package.json

```diff
@@ -17,7 +17,8 @@
     "ci-check": "npm-run-all prettier:diff --parallel lint tsc",
```

Comment:
We have `ci-check`. Let's just use that in README?

Reply:
We could do that but, The `ci-check` lists the prettier error and doesn't do auto fixing. But in the check-all we will prettify the code.

Reply:
We just need to have something that makes sense locally (so, autofix etc), and something that makes sense for CI (so, fail on mismatch) and have them be consistent.

Reply:
Then lets go with check-all itself. And we shall run prettier auto fixing and linting with auto fix too.